integreat_cms/cms/models/users/organization.py

```diff
@@ -43,6 +42,8 @@ class Organization(AbstractBaseModel):
         default=timezone.now, verbose_name=_("creation date")
     )
 
+    website = models.URLField(max_length=250, verbose_name=_("website"), default="")
```

Comment:
```suggestion
    website = models.URLField(max_length=250, verbose_name=_("website"))
```
It's similar for the icon field - at least, this is a CharField, so an empty string might be ok as temporary default for the existing entries, but I would not leave it in the model class.